fapolicy_analyzer/ui/main_window.py

```diff
@@ -107,6 +125,17 @@ def __set_trustDbMenu_sensitive(self, sensitive):
         menuItem = self.get_object("trustDbMenu")
         menuItem.set_sensitive(sensitive)
 
+    def __update_fapd_status(self, bStatus: bool):
+        logging.debug(f"__update_fapd_status({bStatus})")
+        if bStatus:
+            self.fapdStatus.set_from_stock(stock_id="gtk-yes", size=4)
+        else:
+            self.fapdStatus.set_from_stock(stock_id="gtk-no", size=4)
+
+    def on_update_daemon_status(self, status: bool):
+        logging.debug(f"on_update_daemon_status({status})")
+        GLib.idle_add(self.__update_fapd_status, status[1])
```

Comment:
You shouldn't have to `GLib` this.  The event should be firing on the UI thread.

Reply:
Done.